pkg/app/http.go

```diff
@@ -346,13 +349,15 @@ func (h *Handler) makeAppJS() []byte {
 	if err := template.
 		Must(template.New("app.js").Parse(appJS)).
 		Execute(&b, struct {
-			Env      string
-			Wasm     string
-			WorkerJS string
+			Env            string
+			Wasm           string
+			WorkerJS       string
+			UpdateInterval string
```

Comment:
From https://developer.mozilla.org/en-US/docs/Web/API/setInterval, a delay is a number in milliseconds.

Here can use:
```go
AutoUpdateInterval int64
```